packages/core/package.json

```diff
@@ -40,6 +40,7 @@
         "ts-node": "10.1.0",
         "typescript": "4.4.3",
         "typescript-standard": "^0.3.36",
+        "wot-thing-description-types": "1.1.0-23-March-2023",
```

Comment:
Not sure if this this dependency is actually needed since `wot-typescript-definitions` depends on `wot-thing-description-types`

see 
https://github.com/w3c/wot-scripting-api/blob/c92f945c24495cbf239cf847d8d96ef8930e8ec9/typescript/scripting-api/package.json#L26